src/localization/tools/config_parser/config_parser.h

```diff
@@ -30,28 +30,28 @@
  * @brief      Class for storing the configuration parameters.
  */
 class ConfigParser {
-   public:
-    ConfigParser() {}
-    bool parse(const std::string &iniFile);
-    bool parseYaml(const std::string &yamlFile);
-    void print() const;
+public:
+  ConfigParser() {}
+  bool parse(const std::string &iniFile);
+  bool parseYaml(const std::string &yamlFile);
+  void print() const;
 
-    std::string path2qu = "";
-    std::string path2ref = "";
-    std::string path2quImg = "";
-    std::string path2refImg = "";
-    std::string imgExt = "";
-    std::string costMatrix = "";
-    std::string costOutputName = "";
-    std::string simPlaces = "";
-    std::string hashTable = "";
-    std::string matchingResult = "matches.MatchingResult.pb";
+  std::string path2qu = "";
+  std::string path2ref = "";
+  std::string path2quImg = "";
+  std::string path2refImg = "";
+  std::string imgExt = "";
+  std::string costMatrix = "";
+  std::string costsOutputName = "costs.MatchingCosts.pb";
```

Comment:
Generally speaking, using default values on which your logic depends is frowned upon. The reason is that if your logic changes, these default values are easy to forget. I would suggest to set the default value to empty and just pass the appropriate values into a constructor that takes just the values you need. This way, when reading the code it should be easy to see which objects are created with which values.